### DISCUSSION TITLE
BIT component not found install using NPM/Yarn

### DISCUSSION THREAD

**Discussion by vishalshah6153** [teambit/bit#5734] [Help]
Using React JS, I created components and exported them to a private organization/scope and it's exported successfully. When I import those components using bit import, they work fine on my local machine but they'll not work on vercel deploy. So, I have tried installing with NPM but it's not working and giving me the following error. 

`````
npm ERR! code E404
npm ERR! 404 Not Found - GET https://node.bit.dev/@bit%2forgname.scopename.meta-tag - Not found
npm ERR! 404 
npm ERR! 404  '@bit/orgname.scopename.meta-tag@*' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/star-rainbow/.npm/_logs/2022-04-25T09_37_16_164Z-debug-0.log
`````

Can someone help me out here? Is there anything wrong I'm doing or something else?

**Comment by vishalshah6153** [comment]
Does anyone have any updates on this issue?

**Reply by Nebras-othman** [reply]
I am new to bit, but I think you need to specify the npm registry in your server, and I am not sure ho you can do that in Vercel , most likely in the container config
